homeworkW12D03/src/ba/bitcamp/homeworkW12D03/task01/CountLettersFromFile.java

```diff
@@ -0,0 +1,139 @@
+package ba.bitcamp.homeworkW12D03.task01;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.concurrent.LinkedBlockingQueue;
+
+/**
+ * 
+ * @author Gordan
+ *
+ *         This class represents one program that counts how many time first
+ *         letter from text file repeats in same text file.
+ */
+public class CountLettersFromFile {
+
+	// Defining variables
+	private static int counter = 1;
+	private static LinkedBlockingQueue<Master> assignments;
+	private static ArrayList<Worker> workers;
+	private static Object lock = new Object();
+
+	public static void main(String[] args) {
+
+		// Initializing list and queue
+		assignments = new LinkedBlockingQueue<>();
+		workers = new ArrayList<>();
+
+		try {
+
+			// Creating reader for reading text file
```

Comment:
Since producer/consumer approach was required, you should read the line in separate thread also, so reading and processing happens in parallel. Current implementation first reads the entire file and than starts processing it.